selfdrive/car/gm/carstate.py

```diff
@@ -30,16 +30,18 @@ def get_powertrain_can_parser(CP, canbus):
     ("LKADriverAppldTrq", "PSCMStatus", 0),
     ("LKATorqueDeliveredStatus", "PSCMStatus", 0),
   ]
-
   if CP.carFingerprint == CAR.VOLT:
     signals += [
       ("RegenPaddle", "EBCMRegenPaddle", 0),
+    ]
+  if CP.carFingerprint in (CAR.VOLT, CAR.ASTRA):
+    signals += [
       ("TractionControlOn", "ESPStatus", 0),
       ("EPBClosed", "EPBStatus", 0),
       ("CruiseMainOn", "ECMEngineStatus", 0),
       ("CruiseState", "AcceleratorPedal2", 0),
     ]
-  elif CP.carFingerprint == CAR.CADILLAC_CT6:
+  if CP.carFingerprint == CAR.CADILLAC_CT6:
```

Comment:
not needed change

Reply:
I made this change for consistency - what was previously just an if/elif is now more complicated due to the things in common between the Volt and the Astra. A series of plain ifs seem to capture it better since the cases aren't all mutually exclusive.